tutorials/hsdk_rt_analysis/README.md

```diff
@@ -0,0 +1,77 @@
+# Holoscan SDK Response-Time Analysis 
+
+This directory contains helpful scripts for timing analysis of Holoscan apps, based on published research regarding the Holoscan SDK (see citation below).
+
+Detailed instructions for how to reproduce the results of the paper, along with the code, can be found in the `artifact` directory.
+
+## Scripts
+
+The scripts in this directory are written in Python and require the `networkx` and `pydot` packages, which can be easily installed with pip. Each script takes a representation of an application graph in the form of a DOT file. An example DOT file, `exampledot.dot`, can be used as a model. The graph defined in the DOT file is a simplified view of a Holoscan application, with two parts. The first section consists of the name of each operator, with its worst-case execution time as an attribute. The second section captures data flow between operators. We assume that there is exactly one operator with no inputs, and exactly one operator with no outputs.   
```

Comment:
The scripts in current directory (`computeWCRT.py` and `runsimulation.py`) are written in Python and require the `networkx` and `pydot` packages, which can be easily installed with pip. 

```bash
pip install networkx
pip install pydot
```
Each script takes a representation of an application graph in the form of a [DOT](https://en.wikipedia.org/wiki/DOT_(graph_description_language)) file. An example DOT file is provided in `exampledot.dot`. The DOT file defines a Holoscan application graph in two parts. The first section consists of the names of the operators, with their worst-case execution times as attributes. The second section captures the connections between operators. 

We assume that there is only one root operator and one leaf operator in the application graph, without losing any generality. More information on why this works for more than one root and leaf can be found in the paper [1].